pkgs/leak_tracker/lib/src/leak_tracking/leak_tracker_model.dart

```diff
@@ -78,6 +78,15 @@ class LeakDiagnosticConfig {
       classesToCollectStackTraceOnDisposal.contains(classname);
 }
 
+/// The default value for number of full GC cycles, enough for a non reachable object to be GCed.
+///
+/// It is pessimistic assuming that user will want to
+/// detect leaks not more often than a second.
+///
+/// Theoretically, 2 should be enough, however it gives false positives
+/// if there is no activity in the application for ~5 minutes.
+const defaultGcCountBuffer = 3;
```

Comment:
I'm not 100% if this is public API or not (whether it's exported from somewhere), but if it is we should mention this in the changelog.